README.md

```diff
@@ -1,7 +1,55 @@
-[![Build Status](https://travis-ci.org/Microsoft/applicationinsights-core-js.svg?branch=master)](https://travis-ci.org/Microsoft/applicationinsights-core-js)
-[![Build status](https://dev.azure.com/mseng/AppInsights/_apis/build/status/AppInsights%20-%20DevTools/1DS%20JavaScript%20SDK%20-%20Core)](https://dev.azure.com/mseng/AppInsights/_build/latest?definitionId=7605)
+# Applicationinsights corejs [![Build Status](https://travis-ci.org/Microsoft/applicationinsights-core-js.svg?branch=master)](https://travis-ci.org/Microsoft/applicationinsights-core-js) [![Build status](https://dev.azure.com/mseng/AppInsights/_apis/build/status/AppInsights%20-%20DevTools/1DS%20JavaScript%20SDK%20-%20Core)](https://dev.azure.com/mseng/AppInsights/_build/latest?definitionId=7605)
```

Comment:
Title on separate line from build badges.
Change title to 'Microsoft Application Insights JavaScript SDK - Core'